pkg/operator/ceph/cluster/mon/health_test.go

```diff
@@ -55,8 +55,9 @@ func TestCheckHealth(t *testing.T) {
 		ConfigDir: configDir,
 		Executor:  executor,
 	}
-	c := New(context, "ns", "", "myversion", cephv1.CephVersionSpec{}, cephv1.MonSpec{Count: 3, AllowMultiplePerNode: true},
-		rookalpha.Placement{}, false, v1.ResourceRequirements{}, metav1.OwnerReference{})
+	c := New(nil, context, "ns", "", "myversion", cephv1.CephVersionSpec{},
+		cephv1.MonSpec{Count: 3, AllowMultiplePerNode: true}, rookalpha.Placement{}, false,
+		v1.ResourceRequirements{}, metav1.OwnerReference{})
 	c.clusterInfo = test.CreateConfigDir(1)
```

Comment:
There are a lot of tests that currently set `clusterInfo` immediately after calling `New()`. Should we pass that to the constructor instead of nil?